compiler/testData/asJava/lightClasses/compilationErrors/TraitClassObjectField.kt

```diff
@@ -4,5 +4,7 @@ interface TraitClassObjectField {
     companion object {
         const val x: String? = ""
         private val y: String? = ""
+            // Custom getter is needed, otherwise no need to generate getY
```

Comment:
Please remove the accessor from the `.java` file in this test instead of this change

Reply:
Done